bpf/tests/wireguard_helpers_tests.c

```diff
@@ -91,7 +91,7 @@ int check2(struct __ctx_buff *ctx)
 
 	assert(!ctx_mark_is_wireguard(ctx));
 
-	ctx->mark = MARK_MAGIC_WG_ENCRYPTED;
+	ctx->mark = MARK_MAGIC_ENCRYPT;
```

Comment:
Should the semantic here still be 'MARK_MAGIC_ENCRYPTED' ? The change you have here makes it sound like the mark indicates the associated SKB should be encrypted, not that it has been encrypted?

Reply:
PR where we moved from MARK_MAGIC_WG_ENCRYPTED to MARK_MAGIC_ENCRYPT https://github.com/cilium/cilium/pull/39651.
The idea is that we could've reused the same, given IPSec and WireGuard are mutually exclusive.
In IPSec, we use it both to indicate packets both pkts that needs to be encrypted and those who've just being encrypted. In Wireguard, we see the mark only after encryption.